src/main/java/edu/kis/powp/jobs2d/command/manager/CommandChangeObserver.java

```diff
@@ -0,0 +1,12 @@
+package edu.kis.powp.jobs2d.command.manager;
+
+import edu.kis.powp.jobs2d.features.CommandsFeature;
+import edu.kis.powp.observer.Subscriber;
+
+public class CommandChangeObserver  implements Subscriber {
```

Comment:
This observer is responsible for updating commands history, so this aspect should be reflected in the class name.